WebRoot/js/zimbraMail/core/ZmOperation.js

```diff
@@ -185,7 +185,7 @@ function() {
 		AjxCallback.simpleClosure(function(parent) {
 			ZmOperation.addDeferredMenu(ZmOperation.addSearchMenu, parent, true);
 		}));
-	ZmOperation.registerOp(ZmId.OP_SEND, {textKey:"send", tooltipKey:"sendTooltip", image:"Send", shortcut:ZmKeyMap.SEND});
+	ZmOperation.registerOp(ZmId.OP_SEND, {textKey:"send", tooltipKey:"sendTooltip", image:"Send", shortcut:ZmKeyMap.SEND, showImageInToolbar: true, showTextInToolbar: true});
```

Comment:
in case of we have preference MAIL_SEND_LATER_ENABLED as true

then we are using ZmId.OP_SEND_LATER operation, does this changes needs to be done there as well?